readme.md

```diff
@@ -83,13 +83,12 @@ This option is passed to [`mdast-util-to-html`][to-mdast-handlers].
 
 ###### `options.sanitize`
 
-How to sanitize the output (`Object` or `boolean`, default: `true`).
+How to sanitize the output (`Object` or `boolean`, default: `true`):
 
-If `false`, no HTML is sanitized, and dangerous HTML is left unescaped.
-
-If `true` or an `object`, sanitation is done by [`hast-util-sanitize`][sanitize]
-If an object is passed in, it’s given as a schema to `hast-util-sanitize`.
-If `true`, input is sanitized according to [GitHub’s sanitation rules][github].
+- If `false`, no HTML is sanitized, and dangerous HTML is left unescaped.
+- If `true` or an `object`, sanitation is done by [`hast-util-sanitize`][sanitize].
+- If an object is passed in, it’s given as a schema to `hast-util-sanitize`.
+- If `true`, input is sanitized according to [GitHub’s sanitation rules][github].
```

Comment:
Great idea!

```suggestion
*   `false`
    — HTML is not sanitized, dangerous HTML persists
*   `true`
    — HTML is [`hast-util-sanitize`][sanitize] according to [GitHub’s sanitation
    rules][github], dangerous HTML is dropped
-   `Object`
    — the object is treated as a `schema` for how to sanitize with
    [`hast-util-sanitize`][sanitize], dangerous HTML is dropped
```

How about this? a) Reworded in a way that makes more sense to me, can you confirm, b) conforms to code style, c) dropped the fourth, duplicate, bullet, which was a bit confusing to me?

Reply:
yes this looks better for me :)